src/vivarium_cluster_tools/psimulate/runner.py

```diff
@@ -177,6 +175,37 @@ def __init__(self, arguments):
         self.model_specification = self.output_directory / 'model_specification.yaml'
 
 
+class NativeSpecification:
+    def __init__(self, project, peak_memory, max_runtime, validation='n', threads=1, job_name='vivarium', **__):
+        self.project = project
```

Comment:
Are we still validating the project somewhere?

Reply:
the click option validates because its type is choice